flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -256,20 +257,31 @@ static void msp_send_comp_gps(struct msp_bridge *m)
 		} else {
 			data.comp_gps.home_position_valid = 1;  // Home distance and direction will display on OSD
 			
-			int32_t delta_lon = (home_data.Longitude - gps_data.Longitude);  // degrees * 1e7
-			int32_t delta_lat = (home_data.Latitude  - gps_data.Latitude );  // degrees * 1e7
+//			int32_t delta_lon = (home_data.Longitude - gps_data.Longitude);  // degrees * 1e7
+//			int32_t delta_lat = (home_data.Latitude  - gps_data.Latitude );  // degrees * 1e7
 	
-			float delta_y = delta_lon * WGS84_RADIUS_EARTH_KM * DEG2RAD;  // KM * 1e7
-			float delta_x = delta_lat * WGS84_RADIUS_EARTH_KM * DEG2RAD;  // KM * 1e7
+//			float delta_y = delta_lon * WGS84_RADIUS_EARTH_KM * DEG2RAD;  // KM * 1e7
+//			float delta_x = delta_lat * WGS84_RADIUS_EARTH_KM * DEG2RAD;  // KM * 1e7
 	
-			delta_y *= cosf(home_data.Latitude * 1e-7f * (float)DEG2RAD);  // Latitude compression correction
-	
-			data.comp_gps.distance_to_home  = (uint16_t)(sqrtf(delta_x * delta_x + delta_y * delta_y) * 1e-4f);  // meters
+//			delta_y *= cosf(home_data.Latitude * 1e-7f * (float)DEG2RAD);  // Latitude compression correction
+
+			float linearized_conversion_factor_f[3];
+			LLA2NED_linearization_float(home_data.Latitude, home_data.Altitude, linearized_conversion_factor_f);
```

Comment:
I'm unsure about how I feel about doing the linearization each loop. @peabody124, what's your opinion about increased code complexity vs. the CPU cost of doing this per-loop? I guess I'm kind of okay giving it a pass, since it's just a few Hz.